vignettes/introduction.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introduction"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Introduction}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+knitr::read_chunk(path = system.file("plumber", "capitalize", "plumber.R", package = "plumbertableau"),
+                  from = 9,
+                  labels = "capitalize")
+```
+
+This package is designed to enable R developers to build [Tableau Analytics Extensions](https://tableau.github.io/analytics-extensions-api/) and deploy those extensions to [RStudio Connect](https://www.rstudio.com/products/connect/). The goal is to make it easy for R developers to build Tableau extensions and to make it easy for Tableau developers to use those extensions.
+
+### R and Tableau
+Tableau is an industry leading visual analytics platform. Many organizations use Tableau to investigate, understand, and report on data. While Tableau offers some analytics capabilities, there are times when it's useful to perform additional analytics within Tableau. For example, it may be desirable to include real time output from a predictive model in a Tableau dashboard. This type of real time reporting can be accomplished using Tableau Analytics Extensions^[See [Announcing the Analytics Extensions API](https://www.tableau.com/about/blog/2020/3/announcing-analytics-extensions-api)].
+
+R is a popular language for statistics and data analytics. Referencing R extensions from within Tableau workbooks provides Tableau developers with the ability to leverage additional functionality provided by the R language. In this way, tools like R and Tableau can be used to compliment one another and provide more comprehensive data insights.
+
+### Implementation
+The `plumbertableau` package is an extension of the [`plumber`](https://www.rplumber.io/) package, which is a package for developing APIs in R. `plumbertableau` extends `plumber` and provides additional features necessary to comply with the [Tableau Analytics API specification](https://tableau.github.io/analytics-extensions-api/docs/ae_api_ref.html). At it's core, developing an analytics extension for Tableau in R is a matter of building an R based API that Tableau can submit requests to. A simple example is creating an API that capitalizes all incoming text:
```

Comment:
Looking at RStudio's other packages, our general style is to _not_ code-style package names, even if they're lower case. That said, some packages, like Plumber, have capitalized names.

Reply:
e.g. https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html

Reply:
I'm gonna make edits to this effect as I read :)